src/hotspot/cpu/ppc/vm_version_ppc.cpp

```diff
@@ -37,7 +37,9 @@
 #include "utilities/globalDefinitions.hpp"
 #include "vm_version_ppc.hpp"
 
+#ifndef __FreeBSD__
```

Comment:
None of the BSD have sys/sysinfo.h that I know of.

Reply:
ok